paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -56,7 +58,12 @@ class PaparazziPlugin : Plugin<Project> {
         .libraryVariants
     variants.all { variant ->
       val variantSlug = variant.name.capitalize(Locale.US)
+      val testVariantSlug = variant.unitTestVariant.name.capitalize(Locale.US)
+
+      val nonTransitiveRClassEnabled = (project.findProperty("android.nonTransitiveRClass") as String?)?.toBoolean() ?: false
 
+      val generateStubRFileProvider = project.tasks.named("generate${testVariantSlug}StubRFile") as Provider<GenerateLibraryRFileTask>
```

Comment:
temporary, until we can use artifact view apis to ask for artifacts of type `SYMBOL_LIST_WITH_PACKAGE_NAME`